squidpy/pl/_var_by_distance.py

```diff
@@ -0,0 +1,176 @@
+from __future__ import annotations
+
+from pathlib import Path
+from types import MappingProxyType
+from typing import Any, List, Mapping, Optional, Sequence, Tuple, Union
+
+import matplotlib.cm as cm
+import matplotlib.colors as colors
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn as sns
+from anndata import AnnData
+from cycler import Cycler
+from matplotlib import rcParams
+from matplotlib.axes import Axes
+from scanpy.plotting._tools.scatterplots import _panel_grid
+from scanpy.plotting._utils import _set_default_colors_for_categorical_obs
+from scipy.sparse import issparse
+
+from squidpy.pl._utils import save_fig
+
+__all__ = ["var_by_distance"]
+
+
+def var_by_distance(
+    adata: AnnData,
+    design_matrix_key: str,
```

Comment:
what's the default here? assign the default that is used in the `tl` function so user can run consecutive these functions.